api/Controllers/MiniUrl.cs

```diff
@@ -48,7 +40,8 @@ public ActionResult<MinifyResponse> Post([FromBody] MinifyRequest body)
         string urlId = Guid.NewGuid().ToString();
         urlId = urlId.Substring(0, 8);
```

Comment:
I'm really not sure, if first 4 bytes of guid are good idea as unique identifier. You can hash URL to get better uniqness and distribution